lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -169,15 +167,18 @@ <h3 class="course-title">
                       href="${facebook_url}"
                       target="_blank"
                       title="${_('Share on Facebook')}"
+                      data-course-id="${course_overview.id}"
                       onclick="window.open('${facebook_url}', '${share_window_name}', '${share_window_config}'); return false;">
                       <span class="sr">${_('Facebook')}</span>
                       <span class="fa fa-facebook" aria-hidden="true"></span>
                     </a>
                 % endif
-                % if share_url and share_settings.get('DASHBOARD_TWITTER', False):
-                  <% share_text_default = _("I'm learning on {platform_name}:").format(platform_name=settings.PLATFORM_NAME) %>
-                  <% share_text = urllib.quote_plus(share_settings.get('DASHBOARD_TWITTER_TEXT', share_text_default)) %>
-                  <% twitter_url = 'https://twitter.com/intent/tweet?text=' + share_text + '%20' + share_url %>
+                % if share_settings.get('DASHBOARD_TWITTER', False):
+                  <%
+                      default_share_text = _("I'm taking {course_name} online with @edxonline. Check it out!").format(course_name=course_overview.display_name_with_default)
+                      share_text = urllib.quote_plus(share_settings.get('DASHBOARD_TWITTER_TEXT', default_share_text.encode('utf-8')))
```

Comment:
I would encode both values (DASHBOARD_TWITTER_TEXT and default_share_text) to avoid future error.
something like:
```
urllib.quote_plus(share_settings.get('DASHBOARD_TWITTER_TEXT', default_share_text).encode())